documents/model.go

```diff
@@ -31,14 +32,23 @@ const (
 	SignaturesField = "signatures"
 	// SigningRootField represents the signature root property of a tree
 	SigningRootField = "signing_root"
+	// CDTreePrefix is the human readable prefix for core doc tree props
+	CDTreePrefix = "cd_tree"
+	// SigningTreePrefix is the human readable prefix for signing tree props
+	SigningTreePrefix = "signing_tree"
 )
 
-var compactProperties = map[string][]byte{
-	CDRootField:       {0, 0, 0, 7},
-	DataRootField:     {0, 0, 0, 5},
-	DocumentTypeField: {0, 0, 0, 100},
-	SignaturesField:   {0, 0, 0, 6},
-	SigningRootField:  {0, 0, 0, 10},
+func compactProperties(key string) []byte {
+	m := map[string][]byte{
+		CDRootField:       {0, 0, 0, 7},
+		DataRootField:     {0, 0, 0, 5},
+		DocumentTypeField: {0, 0, 0, 100},
+		SignaturesField:   {0, 0, 0, 6},
+		SigningRootField:  {0, 0, 0, 10},
+		CDTreePrefix:      {0, 0, 0, 11},
+		SigningTreePrefix: {0, 0, 0, 12},
```

Comment:
@rdinicut the prefixes and their keys CDTreePrefix and SigningTreePrefix

Reply:
This could also be a candidate to move to the other repo?

Reply:
A couple of questions come up for me:
1. Do we need 4 bytes here? Or we can get away with it with let's say 2.
2. Should we follow a different pattern between prefixes and fields? invoice/po are `{x,0,0,0}` vs CDTree and SigningTreePrefix `{0,0,0,x}`.
I am fine renaming invoice/po or these ones, but would be better to keep it consistent.

@vimukthi-git @lucasvo thoughts?

Reply:
We could switch two two bytes, though I wouldn't do less. I think @vimukthi-git picked 32 because that's the standard field length. Though we are not required to oblige to it at all.

Reply:
@lucasvo yes, I have no opinion on this, just using uint32.
@mikiquantum I can make that change.